Tasks/ServiceFabricDeploy/Create-DiffPackage.psm1

```diff
@@ -148,6 +148,11 @@ function Copy-DiffPackage
             $localPkgPath += ".zip"
             $diffPkgPath += ".zip"
         }
+        # The Code package for containerized service does not exist, but we want to continue the deployment
+        elseif (!(Test-Path -Path $localPkgPath))
```

Comment:
This ignores the scenario when package does not exist when service is non-containerized. Do we want to not throw in such case?
This should also be fine as we do a validation downstream using Test-ServiceFabricApplicationPackage - which anyways would fail
However, would it be better to move Test-ServiceFabricApplicationPackage before doing diff'ing as well. This will ensure that we do have a valid package before diffing - giving us more confidence that diffing has not invalidated the package.

Reply:
@bishal-pdMSFT Added Test-ServiceFabricApplicationPackage before creating diff package in deploy.ps1

Reply:
@bishal-pdMSFT Do you have any other concern on this PR?